src/overlays/actors/ovl_Obj_Bean/z_obj_bean.h

```diff
@@ -6,9 +6,38 @@
 
 struct ObjBean;
 
+typedef void (*ObjBeanActionFunc)(struct ObjBean*, GlobalContext*);
+
+typedef void (*ObjBeanTransformFunc)(struct ObjBean*);
+
+
```

Comment:
id remove extra beaks here. group the prototypes together and have one break before the struct